src/App.vue

```diff
@@ -68,6 +68,13 @@ export default {
       }else{
         this.todoList[todoId].todoState = '作業中';
       }
+    },
+    displayPermit: function(checkDisplayState,todoStatus){
```

Comment:
関数は以下のように省略して記述できます。
```suggestion
    displayPermit(checkDisplayState,todoStatus){
```